ui/src/common/components/__tests__/IncomingLiteratureReferencesLinkAction.test.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import IncomingLiteratureReferencesLinkAction from '../IncomingLiteratureReferencesLinkAction';
+
+describe('IncomingLiteratureReferencesLinkAction', () => {
+  it('renders with required props', () => {
+    const itemCount = 29;
+    const displayedName = 'citation';
+    const recordId = 888;
+    const linkQuery = `refersto:recid:${recordId}`;
+    const wrapper = shallow(
+      <IncomingLiteratureReferencesLinkAction
+        itemCount={itemCount}
+        linkQuery={linkQuery}
+        displayedName={displayedName}
```

Comment:
Also, I would rename this to ```referenceType``` as Harun suggested. It goes better with the name of the component